src/dispatch_egl.c

```diff
@@ -62,12 +62,16 @@ epoxy_egl_version(EGLDisplay dpy)
 {
     int major, minor;
     const char *version_string;
-    int ret;
+    int ret=0, sscanf_ret;
 
-    version_string = eglQueryString(dpy, EGL_VERSION);
-    ret = sscanf(version_string, "%d.%d", &major, &minor);
-    assert(ret == 2);
-    return major * 10 + minor;
+    if ((version_string = eglQueryString(dpy, EGL_VERSION)))
```

Comment:
The coding style is also off, here.

It would be great to replace the whole block with a simple NULL check, e.g.:

````
    version_string = eglQueryString(dpy, EGL_VERSION);
    if (!version_string)
        return 0;
```